src/features/search/components/Highlight/Highlight.tsx

```diff
@@ -22,14 +22,30 @@ export const HighlightPart: FunctionComponent<HighlightPartProps> = ({
   )
 }
 
-type HighlightProps = {
-  hit: AlgoliaSuggestionHit
+type WithSuggestionHitProps = {
+  suggestionHit: AlgoliaSuggestionHit
+  venueHit?: never
+}
+
+type WithVenueHitProps = {
+  venueHit: AlgoliaVenue
+  suggestionHit?: never
+}
+
+type HighlightProps = (WithSuggestionHitProps | WithVenueHitProps) & {
   attribute: string
 }
 
-export const Highlight: FunctionComponent<HighlightProps> = ({ hit, attribute }) => {
-  const { value } = getPropertyByPath(hit._highlightResult, attribute) || {}
-  const attributeValue = value.toString()
+export function Highlight({ suggestionHit, venueHit, attribute }: HighlightProps) {
+  let attributeValue = ''
+  if (suggestionHit) {
+    const { value } = getPropertyByPath(suggestionHit._highlightResult, attribute) || {}
+    attributeValue = value.toString()
```

Comment:
C'est bizarre car in peut potentiellement avoir value === undefined
Que retourne undefined.toString() ?

Reply:
La fonction ne renverra jamais `undefined` à partir du moment où tu as une suggestion

Reply:
Ok mais alors pourquoi écrire || {} ?
On est d'accord que const { value } = {} ça donne value <- undefined ?

Reply:
Yes tu as raison je peux le retirer